.github/workflows/e2e.yml

```diff
@@ -0,0 +1,48 @@
+# e2e test for Wp-menu-custom-fields
+
+name: CI for rtsocial Plugin Test cases
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [develop]
```

Comment:
@alvitazwar 
The branches should be develop and master similar to https://github.com/rtCamp/wp-menu-custom-fields/blob/594beed7511097aa2d58b95a5701b261348ef6fe/.github/workflows/e2e.yml#L9